packages/hub/src/spancontext.ts

```diff
@@ -0,0 +1,47 @@
+import { SpanContext as SpanContextInterface } from '@sentry/types';
+import { uuid4 } from '@sentry/utils';
+
+export const TRACEPARENT_REGEX = /([0-9a-f]{2})-([0-9a-f]{32})-([0-9a-f]{16})-([0-9a-f]{2})/;
```

Comment:
`REGEXP`

Reply:
It should most likely also include `/^$/` so it only matches a whole thing

Reply:
I took this regex from Armin out of python, haven't really thought about it.